### PR TITLE
Fixed #1930 Added Fade Filter

### LINKED ISSUE #1930: Add Fade  BW contrast filter.
**Actual Behaviour**

Add Fade filter.
Fade filter will allow increase or decrease in black-white contrast.

**Would you like to work on the issue?**

yes

### PR DESCRIPTION
**Pull request by Pravesh-Jamgade**
Fixed https://github.com/fossasia/phimpme-android/issues/1930

**Changes**:
I added fade filter.It creates greyscale image and perform bw contrast.
I changed files from **phimpme-android/app/src/main/jni/** location containing filters.cpp   ,filters.h , main_processing.cpp and an array file from **app/src/main/res/values/arrays.xml** .

**Screenshots for the change**:
**Before** 
image
**After** 
image

### CODE REVIEW

**Comment by Pravesh-Jamgade** [comment]
@mayank8318  earlier i really messed up with my branches.Please review now.I hope these time i done everything correctly.

File: app/src/main/jni/filters.cpp [hunk 1]
```diff
@@ -534,5 +535,23 @@ void applyCyano(cv::Mat &src, cv::Mat &dst, int val) {
     }
 }
 
+void applyFade(cv::Mat &src, cv::Mat &dst, int val)
```
Comment (codedsun) [inline app/src/main/jni/filters.cpp]: Please follow the coding style and update the PR.

File: app/src/main/jni/filters.cpp [hunk 2]
```diff
@@ -534,5 +535,23 @@ void applyCyano(cv::Mat &src, cv::Mat &dst, int val) {
...
+{
+    cvtColor(src,src,COLOR_RGB2GRAY);
+    dst = src.clone();
+    for(int y=0 ;y < src.rows ; y++)
+    {
```
Comment (codedsun) [inline app/src/main/jni/filters.cpp]: same

**Comment by Pravesh-Jamgade** [comment]
@codedsun  done

**Pull request merged**